llvm/test/Transforms/LoopUnroll/AMDGPU/unroll-dependent-sub.ll

```diff
@@ -0,0 +1,84 @@
+; RUN: opt -S -mtriple=amdgcn-- -passes=loop-unroll -debug-only=AMDGPUtti < %s 2>&1 | FileCheck %s
```

Comment:
Better to check the actual output.

If you're going to check the debug output, might as well use -debug. But you also need REQUIRES: asserts